client/containers/EventsPanelContainer/index.jsx

```diff
@@ -29,9 +29,10 @@ EventsPanel.propTypes = {
 
 const mapStateToProps = (state) => ({
     // selectors.getShowEventDetails is either true, an event id, or null
-    eventToDetail: selectors.getShowEventDetails(state) === true
-        ? DEFAULT_FORM_VALUES
-        : selectors.getEventToBeDetailed(state),
+    eventToDetail: selectors.getShowEventForm(state) ? selectors.getShowEventDetails(state) === true
```

Comment:
Hmmm.. yes, that was my first approach. But the event has no details about "_plannings" in the event object. "_plannings" are required to show the related plannings of an event to delete it. You can take a look at the test instance (http://sdppr-136.test.superdesk.org/), by adding an event to an agenda and deleting the event. I will give it another look to see if I'm missing anything and try not to touch showEventDetail :)